src/components/Button/Button.tsx

```diff
@@ -93,16 +95,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         variant={variant}
         {...other}
       >
-        {loading && (
-          <Icon
-            className={styles['eds-is-loading']}
-            name="spinner"
-            purpose="informative"
-            size="1.5em"
-            title="loading"
-          />
-        )}
-
+        {loading && <LoadingIndicator className="eds-is-loading" size="sm" />}
```

Comment:
When do you think this variant will be removed? Just not a fan of this `eds-is-loading` classnaming haha should be more BEM

Reply:
Let's chat about it this week. we could do whenever, but removing is technically breaking, so didn't want to commit to a v12 just yet :) 

agreed on that class and this implementation 🪓